package_build/initialize.sh

```diff
@@ -28,6 +28,7 @@ do
   ln -sf $BASEDIR/dist $BASEDIR/package_build/$CONFIG_NAME/
   ln -sf $BASEDIR/tfx $BASEDIR/package_build/$CONFIG_NAME/
   ln -sf $BASEDIR/README*.md $BASEDIR/package_build/$CONFIG_NAME/
+  ln -sf $BASEDIR/LICENSE $BASEDIR/package_build/$CONFIG_NAME/
```

Comment:
Do we intend to remove this file in a future revision of the build system? It's unusual to be linking in a bunch of files and directories as part of a build step, and I'm wondering if there's a better way.

OTOH if this is just temporary until the rest of the build system rework can be done I'm on board.

Reply:
>Do we intend to remove this file in a future revision of the build system? It's unusual to be linking in a bunch of files and directories as part of a build step, and I'm wondering if there's a better way.

I agree with you and I would prefer to get rid of it as well, but the problem is we're building two packages (one subset of other) with the same source code and there is no way to add multiple `pyproject.toml`(s) in the root directory (or I am not aware of any), hence this linking. So, until we find an alternative to achieve this, unfortunately we have to keep this.